### DISCUSSION TITLE
CNCF Sandbox onboarding

### DISCUSSION THREAD

**Discussion by rootfs** [sustainable-computing-io/kepler#695] [Announcements]
If you are in any of these teams, you are a Kepler maintainer.
@sustainable-computing-io/community-manager @sustainable-computing-io/community-manager @sustainable-computing-io/maintainer @sustainable-computing-io/reviewer @sustainable-computing-io/kepler-helm-maintainer @sustainable-computing-io/kepler-deployment 

- Please review the onboarding process [here](https://github.com/cncf/toc/issues/1054)
- Please provide your info in this format: Name, Company, Github ID, Email.
- Please volunteer the required tasks

Please do it with urgency. Let's finish the onboarding process in two weeks by June 2nd.

# Tasks:
From the project side, please ensure that you:

- [x] Understand the project proposal process and reqs: https://github.com/cncf/toc/blob/main/process/project_proposals.md#introduction
- [x] Understand the services available for your project at CNCF https://www.cncf.io/services-for-projects/
- [x]  Ensure your project meets the CNCF IP Policy: https://github.com/cncf/foundation/blob/master/charter.md#11-ip-policy
- [x]  Review the online programs guidelines: https://github.com/cncf/foundation/blob/master/online-programs-guidelines.md
- [x] Understand the trademark guidelines: https://www.linuxfoundation.org/en/trademark-usage/
- [x] Understand the license allowlist: https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist
- [ ] Is your project working on written, open governance? see https://contribute.cncf.io/maintainers/governance/
- [ ]  Slack: Are your slack channels migrated to the Kubernetes or CNCF Slack? (see https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another for more details)
- [x]  Is your project in its own separate neutral github organization?
- [ ]  Submitted a Pull request to add your project as a sandbox project to https://landscape.cncf.io/
- [ ]  Create maintainer list + add to aggregated https://maintainers.cncf.io/ list by submitting a PR to it
- [ ]  Artwork: Submit a pull request to https://github.com/cncf/artwork with your artwork
- [x]  Domain: transfer domain to the CNCF - https://jira.linuxfoundation.org/plugins/servlet/theme/portal/2/create/63

Things that CNCF will need from the project:

- [ ]  Provide emails for the maintainers added to https://maintainers.cncf.io/ in order to get access to the maintainers mailing list and ServiceDesk - [project-onboarding@cncf.io](mailto:project-onboarding@cncf.io) is the best email to send those to
- [ ]  Trademarks: transfer any trademark and logo mark assets over to the LF - https://github.com/cncf/foundation/tree/master/agreements has agreements
- [x]  GitHub: ensure 'thelinuxfoundation' and 'caniszczyk' are added as initial org owners, this helps us make sure we have continuity of GH ownership that we will onboard to our GitHub Enterprise instance: https://github.com/enterprises/cncf
- [ ]  GitHub: ensure [DCO](https://github.com/apps/dco) or [CLA](https://github.com/cncf/cla) are enabled for all GitHub repositories of the project
- [ ]  GitHub: ensure that that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) (or your adopted version of it) are explicitly referenced at the project's README on GitHub
- [ ]  Website: ensure LF footer is there and [website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) followed (if your project doesn't have a dedicated website, please adopt those guidelines to the README file of your project on GitHub). @nikimanoledaki @husky-parul can you help this one?
- [ ]  Website: Analytics transferred to [projects@cncf.io](mailto:projects@cncf.io)
- [ ]  OpenSSF Best Practices Badge: Start on an OpenSSF Best Practices Badge https://bestpractices.coreinfrastructure.org/en

# Maintainers:

|  Name  |  Company  | Github ID  | Email   |
|---|---|---|---|
| Huamin Chen  | Red Hat  | rootfs  | hchen@redhat.com  |
| Ji Chen | IBM | jichenjc | jichenjc@cn.ibm.com |
| Parul Singh | Red Hat | husky-parul | parsingh@redhat.com |
| Kaiyi Liu | Red Hat | KaiyiLiu1234 | kaliu@redhat.com |
| Peng Hui Jiang | IBM | jiangphcn | jiangph@cn.ibm.com |
| William Caban | Red Hat | williamcaban | wcabanba@redhat.com |
| Yi Yuan | IBM | SamYuan1990 | yy19902439@126.com |
| Qi Feng Huo | IBM | huoqifeng | huoqif@cn.ibm.com |
| Ken Lu | Intel | kenplusplus | ken.lu@intel.com |
| Marcelo Amaral | IBM | marceloamaral | marcelo.amaral1@ibm.com |
| Niki Manoledaki | Weaveworks | nikimanoledaki | niki@weave.works |
| Brad McCoy | Basiq | bradmccoydev | bradmccoydev@gmail.com |
| Sunyanan Choochotkaew | IBM |  sunya-ch | sunyanan.choochotkaew1@ibm.com | 
| Ruomeng Hao | Intel | ruomengh | ruomengh@intel.com | 
Chen Wang | IBM | wangchen615 | chen.wang1@ibm.com |

**Comment by jichenjc** [comment]
Ji Chen, IBM , jichenjc , [jichenjc@cn.ibm.com](mailto:jichenjc@cn.ibm.com)

**Reply by rootfs** [reply]
put it here so I can compile and send to [project-onboarding@cncf.io](mailto:project-onboarding@cncf.io)

**Reply by jichenjc** [reply]
Ji Chen,  IBM , jichenjc , jichenjc@cn.ibm.com

**Reply by husky-parul** [reply]
Parul Singh, Red Hat, husky-parul parsingh@redhat.com

**Reply by kaiyiliu-strala** [reply]
Kaiyi Liu, Red Hat, KaiyiLiu1234, kaliu@redhat.com

**Comment by jiangphcn** [comment]
Peng Hui Jiang, IBM, jiangphcn, jiangph@cn.ibm.com

**Comment by williamcaban** [comment]
William Caban, Red Hat, williamcaban, wcabanba@redhat.com

**Comment by SamYuan1990** [comment]
Yi Yuan, IBM, @SamYuan1990 , [yy19902439@126.com](mailto:yy19902439@126.com)

**Comment by huoqifeng** [comment]
Qi Feng Huo, IBM, huoqifeng, huoqif@cn.ibm.com

**Comment by kenplusplus** [comment]
Ken Lu, Intel, ken.lu@intel.com

**Comment by marceloamaral** [comment]
Marcelo Amaral, IBM, marcelo.amaral1@ibm.com

**Comment by nikimanoledaki** [comment]
Niki Manoledaki, Weaveworks, nikimanoledaki, niki@weave.works

**Comment by bradmccoydev** [comment]
Brad McCoy, Basiq, bradmccoydev, bradmccoydev@gmail.com

**Comment by sunya-ch** [comment]
Sunyanan Choochotkaew, IBM, sunya-ch, sunyanan.choochotkaew1@ibm.com

**Comment by ruomengh** [comment]
Ruomeng Hao, Intel, ruomengh, ruomengh@intel.com

**Comment by wangchen615** [comment]
Chen Wang, IBM, wangchen615, chen.wang1@ibm.com

**Comment by rootfs** [comment]
>  Website: ensure LF footer is there and [website guidelines](https://github.com/cncf/foundation/blob/master/website-guidelines.md) followed (if your project doesn't have a dedicated website, please adopt those guidelines to the README file of your project on GitHub). 

@nikimanoledaki @husky-parul can you help this one?

**Reply by nikimanoledaki** [reply]
sure, I have some time next week! @husky-parul, we can sync on slack if you want :)

**Reply by rootfs** [reply]
thank you! If possible, can you add cncf website analytics too (I don't think we have any right now)

> Website: Analytics transferred to [projects@cncf.io](mailto:projects@cncf.io)

**Reply by nikimanoledaki** [reply]
anytime! it would help to separate the original list above into new issues to track them. this seems like an epic to me 🤔 
https://github.com/sustainable-computing-io/kepler/issues/new?created_from_discussion_number=695

**Comment by eklee15** [comment]
Eun Kyung Lee, IBM , eklee15 , eunkyung.lee@us.ibm.com
